### ISSUE TITLE
Browser Refresh and Index page

### ISSUE THREAD

**Issue by djmtype** [BetterBrandAgency/betterboilerplate#3]
Is there an automatic browser refresh task? 

Also, coming from some other Gulp boilerplates like Sage and Yeoman Generator Webapp, it was confusing at first where I should add  HTML and PHP pages. On the next update, please consider adding a default HTML page that includes the correct CSS and JS paths.

Thanks for the awesome hard work ;)

**Comment by matchboxhero** [comment]
Hi @djmtype,

Thanks for the feedback. There isn't but if this is something users want I can certainly add it in.

That's a good suggestion, I know a couple of people have been thrown by that. The initial thinking was that the boilerplate would be added to a new project folder (where I don't know the dev environment) so a HTML file would not be needed, it could being used on a VueJS project or on a WordPress site. However it's probably easier to just delete the file when it's not needed in a particular project.
TLDR; I'll add an index file 😉 

Thanks,
Ste.
